Projections/hydro_MacProjector.cpp

```diff
@@ -212,6 +212,7 @@ MacProjector::setLevelBC (int amrlev, const MultiFab* levelbcdata, const MultiFa
                                      "setDomainBC must be called before setLevelBC");
     m_linop->setLevelBC(amrlev, levelbcdata, robin_a, robin_b, robin_f);
     m_needs_level_bcs[amrlev] = false;
+    m_needs_init = true; // Solver is not safe for reuse with Robin BC
```

Comment:
Maybe we should only do this when the robin_* pointers are non null.

Reply:
Simply checking on robin_a should work right? MLABecLap will fail if only robin_a is non-null, right?

Reply:
Yes, I think so.